src/families/stellar/js-getTransactionStatus.ts

```diff
@@ -32,86 +42,170 @@ const getTransactionStatus = async (
   warnings: Record<string, Error>;
   estimatedFees: BigNumber;
   amount: BigNumber;
+  maxAmount: BigNumber;
   totalSpent: BigNumber;
 }> => {
   const errors: Record<string, Error> = {};
   const warnings: Record<string, Error> = {};
   const useAllAmount = !!t.useAllAmount;
 
+  const destinationNotExistMessage =
+    new NotEnoughBalanceBecauseDestinationNotCreated("", {
+      minimalAmount: "1 XLM",
```

Comment:
Don't you want to use a variable instead of hardcoding 1? (maybe BASE_RESERVE if it's related?)